### PR TITLE
enh(logs): insert change logs for /configuration/hosts/severities

### PR DESCRIPTION
**Pull request by selfakiri**
## Description

Add change logs while using API for /configuration/hosts/severities (add/delete and update)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x
- [x] 24.04.x
- [x] 24.10.x
- [x] master

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/src/Core/HostSeverity/Infrastructure/Repository/DbWriteHostSeverityActionLogRepository.php [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+            $actionLog = new ActionLog(
+                ActionLog::OBJECT_TYPE_HOST_SEVERITY,
+                $hostSeverityId,
+                $hostSeverity->getName(),
+                ActionLog::ACTION_TYPE_DELETE,
+                $this->contact->getId()
+            );
+            $this->writeActionLogRepository->addAction($actionLog);
+        } catch (\Throwable $ex) {
+            $this->error($ex->getMessage(), ['trace' => $ex->getTraceAsString()]);
```
Comment (Drago37) [inline centreon/src/Core/HostSeverity/Infrastructure/Repository/DbWriteHostSeverityActionLogRepository.php:88]: log powa ! 
```suggestion
            $this->error("Error while deleting host severity : $ex->getMessage()", ['hostSeverity' => $hostSeverity, 'trace' => $ex->getTraceAsString()]);
```

File: centreon/src/Core/HostSeverity/Infrastructure/Repository/DbWriteHostSeverityActionLogRepository.php [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+            );
+            $actionLogId = $this->writeActionLogRepository->addAction($actionLog);
+            $actionLog->setId($actionLogId);
+
+            $details = $this->getHostSeverityPropertiesAsArray($hostSeverity);
+            $this->writeActionLogRepository->addActionDetails($actionLog, $details);
+            
+            return $hostSeverityId;
+        } catch (\Throwable $ex) {
+            $this->error($ex->getMessage(), ['trace' => $ex->getTraceAsString()]);
```
Comment (Drago37) [inline centreon/src/Core/HostSeverity/Infrastructure/Repository/DbWriteHostSeverityActionLogRepository.php:116]: log powa ! 
```suggestion
            $this->error("Error while adding host severity : $ex->getMessage()", ['hostSeverity' => $hostSeverity, 'trace' => $ex->getTraceAsString()]);
```

File: centreon/src/Core/HostSeverity/Infrastructure/Repository/DbWriteHostSeverityActionLogRepository.php [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+                ActionLog::ACTION_TYPE_CHANGE,
+                $this->contact->getId()
+            );
+
+            $actionLogId = $this->writeActionLogRepository->addAction($actionLog);
+            $actionLog->setId($actionLogId);
+            $this->writeActionLogRepository->addActionDetails($actionLog, $diff);
+
+        } catch (\Throwable $ex) {
+            $this->error($ex->getMessage(), ['trace' => $ex->getTraceAsString()]);
```
Comment (Drago37) [inline centreon/src/Core/HostSeverity/Infrastructure/Repository/DbWriteHostSeverityActionLogRepository.php:204]: log powa ! 
```suggestion
            $this->error("Error while updating host severity : $ex->getMessage()", ['hostSeverity' => $hostSeverity, 'trace' => $ex->getTraceAsString()]);
```

**Review by dmyios — APPROVED** [review]

File: centreon/src/Core/HostSeverity/Infrastructure/Repository/DbWriteHostSeverityActionLogRepository.php (hunk 1 above)
Comment (Drago37) [reply centreon/src/Core/HostSeverity/Infrastructure/Repository/DbWriteHostSeverityActionLogRepository.php]: good !

File: centreon/src/Core/HostSeverity/Infrastructure/Repository/DbWriteHostSeverityActionLogRepository.php (hunk 2 above)
Comment (Drago37) [reply centreon/src/Core/HostSeverity/Infrastructure/Repository/DbWriteHostSeverityActionLogRepository.php]: good !

**Review by Drago37 — APPROVED** [review]

**Review by dmyios — APPROVED** [review]

**Review by Drago37 — APPROVED** [review]

**Pull request merged**